src/Impostor.Plugins.Example/Handlers/PlayerEventListener.cs

```diff
@@ -111,5 +116,11 @@ public void OnPlayerVentEvent(IPlayerVentEvent e)
         {
             _logger.LogInformation("Player {player} vented to {vent}", e.PlayerControl.PlayerInfo.PlayerName, e.NewVent.Name);
         }
+
+        [EventListener]
+        public void t(IPlayerCompletedTaskEvent e)
+        {
+            _logger.LogInformation("Player {player} completed {task},{type},{category} visual {visual}", e.ClientPlayer.Character.PlayerInfo.PlayerName, e.Task.Task.Name, e.Task.Task.Type, e.Task.Task.Category, e.Task.Task.IsVisual);
```

Comment:
Please fix spacing after commas.
Change `e.ClientPlayer.Character` to `e.PlayerControl`